ckanext/recombinant/controller.py

```diff
@@ -182,6 +184,42 @@ def template(self, dataset_type, lang, owner_org):
             abort(404, _('Not found'))
 
         book = excel_template(dataset_type, org)
+
+        if request.method == 'POST':
+            filters = {}
+            for r in dataset['resources']:
+                if (r['name'] == dataset_type):
```

Comment:
Don't use () on if conditions that fit on a single line

This looks like we can't edit data from the 2nd resource.. ?